src/definitions/elements/list.less

```diff
@@ -218,6 +218,16 @@ ol.ui.list ol li,
   color: @itemLinkIconColor;
 }
 
+.ui.list .list > a.item > i.icons,
+.ui.list > a.item > i.icons {
+  color: @lightTextColor;
+}
+
+.ui.list .list > a.item:hover > .icons,
+.ui.list > a.item:hover > .icons {
```

Comment:
Also here. This can be moved to the existing hover declaration in the file so it becomes

```css
/*-------------------
        Hover
--------------------*/
.ui.list .list > a.item:hover > .icons,
.ui.list > a.item:hover > .icons,
.ui.list .list > a.item:hover > .icon,
.ui.list > a.item:hover > .icon {
  color: @itemLinkIconHoverColor;
}
```